content/blog/gain-platform-superpowers-with-the-kebap-stack/index.md

```diff
@@ -0,0 +1,704 @@
+4. Pulumi provisions the supporting infrastructure and bootstraps the running application platform.
+5. Backstage offers a developer portal for service management and discovery.
+6. External Secrets Operator ensures secure and consistent secrets management across different environments.
+
+All components are version-controlled in Git, ensuring auditability and traceability.
+
+Let's see how we bootstrap the whole stack:
+
+##### Ignite the Control Plane cluster with Pulumi
+
+<img src="ignite.png" width="500px">
+
+We use Pulumi code to create the control plane cluster in our cloud provider of choice. After the creation of the
+control plane, we install the Argo CD in a very bare-bones way:
+
+```typescript
+const argocd = new ArgoCD("argocd", {
+    initialObjects: initialObjects
+}, {
+    providers: {
+        kubernetes: k8sProvider,
+    },
+});
+```
+
+The idea is to let then Argo CD fetch the rest of the configuration from the Git repository and apply it to the cluster.
+And here comes the power of GitOps: It will also manage the Argo CD itself. This way, we separated the workload running
+on the cluster from the creation of the cluster itself.
+
+##### Ordering a new development Kubernetes cluster
+
+<img src="order.png" width="500px">
+
+We use Backstage to order a new development Kubernetes cluster. As we want to keep the costs low for development
+cluster, we opted here for the [vCluster](https://www.vcluster.com/) project. The vCluster project allows us to create a
+Kubernetes cluster on demand, which runs as containers on the control plane cluster. The moment the cluster is created,
+we add it to the Argo CD as spoke cluster. This way, we can roll out applications to the development cluster in the same
+way as we would do for production clusters.
+
+See the definition of the `ClusterPolicy` object in the [Kyverno](#bonus-kyverno) section.
+
+##### Ordering a new production Kubernetes cluster
+
+<img src="order_cluster.png" width="500px">
+
+We use Backstage to order a new production Kubernetes cluster. This time, we use the Pulumi code to create the cluster
+in the cloud provider of choice. After the creation of the cluster, we add it to the Argo CD as a production cluster.
+This way, we can roll out applications to the production cluster in the same way as we would do for development
+clusters.
+
+##### Ordering Other Infrastructure
+
+<img src="order_rest.png" width="500px">
+
+We use Backstage to order other infrastructure components, such as databases, message queues, and storage. We use again
+the Pulumi Kubernetes Operator to create the infrastructure components in the Kubernetes cluster. This way, we can
+manage all the infrastructure components in the same way as we manage the Kubernetes clusters.
+
+## Conclusion and Pulumi Cloud as an Off-the-Shelf Solution
+
+Many organizations are adopting a CNCF-based stack for their infrastructure needs. These teams require robust platforms
```

Comment:
Yeah this is more the messaging I was looking for when I made my comments about your intro